src/App.js

```diff
@@ -1,16 +1,15 @@
 import { useEffect } from 'react';
 import { useDispatch } from 'react-redux';
-import { getRockets } from './redux/rockets/rocketsSlice';
+import { getMissions } from './redux/missions/missionSlice';
 import Router from './Router';
 import Navbar from './components/Navbar';
 
 function App() {
   const dispatch = useDispatch();
 
   useEffect(() => {
-    dispatch(getRockets());
```

Comment:
Hi @mrkamin, please, notice that you are deleting getRockets() dispatch.

Try adding both of them:
```
useEffect(() => {
  dispatch(getRockets());
  dispatch(getMissions());
},[dispatch]);
```